stripe/src/test/java/com/stripe/android/EphemeralKeyManagerTest.java

```diff
@@ -250,4 +251,48 @@ public void updateKeyIfNecessary_whenReturnsError_setsExistingKeyToNull() {
         verifyNoMoreInteractions(mKeyManagerListener);
         assertNull(keyManager.getEphemeralKey());
     }
+
+    @Test
+    public void triggerCorrectErrorOnInvalidRawKey() {
+
+        mTestEphemeralKeyProvider.setNextRawEphemeralKey("Not_a_JSON");
+        EphemeralKeyManager<CustomerEphemeralKey> keyManager = new EphemeralKeyManager(
+                mTestEphemeralKeyProvider,
+                mKeyManagerListener,
+                TEST_SECONDS_BUFFER,
+                null,
+                CustomerEphemeralKey.class);
+
+        verify(mKeyManagerListener, times(0)).onKeyUpdate(
+                (CustomerEphemeralKey) isNull(), (String) isNull(), (Map<String, Object>) isNull());
```

Comment:
I think this is better:

`any(CustomerEphemeralKey.class), anyString(), ArgumentMatchers.<String, Object>anyMap()`

Reply:
So I initially tried that, but it fails to error when `onKeyUpdate` is called with nulls

Reply:
don't we expect it not to be called though? so I would assume "never with any" makes more logical sense than "never with nulls"

Reply:
Ah yes, but if I change to the `any` and re-introduce the bug where `onKeyUpdate` is called, the test still passes